locale/fof-follow-tags.yml

```diff
@@ -0,0 +1,41 @@
+fof-follow-tags:
+  forum:
+    settings:
+      notify_new_discussion_label: Iemand start een discussie in een tag die ik volg
+      notify_new_post_label: Iemand post in een tag die ik volg
+
+
+    index:
+      follow_tag_button: Volgen
+      unfollow_tag_button: Stop Volgen
+
+
+    sub_controls:
+      follow_button: => fof-follow-tags.ref.follow
+      following_button: => fof-follow-tags.ref.following
+      following_text: Geattendeerd worden op alle nieuwe discussies. 
+      hiding_button: => fof-follow-tags.ref.hiding
+      hiding_text: Verberg discussie op Alle Discussies pagina.
+      ignoring_button: => fof-follow-tags.ref.ignoring
+      ignoring_text: Geen notificatie.
+      lurking_button: => fof-follow-tags.ref.lurking
+      lurking_text: Notificaties van alle nieuwe discussies en reacties.
+      not_following_button: Niet volgen
+      not_following_text: "Notificaties alleen wanneer @mentioned."
+      notify_alert_tooltip: Ontvang een forum notificatie wanneer er nieuwe discussies of reacties zijn.
+      notify_email_tooltip: Ontvang een email wanneer en nieuwe discussies of reacties zijn.
+
+    notifications:
+      new_discussion_text: "{username} start"
+      new_post_text: "{username} post"
+
+  email:
+    newDiscussionInTag: "[Tag | New Discussion] title"
+    newPostInTag: "[Tag | New Post] title"
+
+  ref:
+    follow: Volgen
+    following: Aan het volgen
+    hiding: Verborgen
+    ignoring: Negeren
+    lurking: Lurken
```

Comment:
["Lurken" has another meaning](https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/lurken). Maybe "loeren", "verschuilen" or some kind would be better

Reply:
Not really. Besides drinking it also means reading without posting [](https://nl.wikipedia.org/wiki/Lurker)

Reply:
Oh alright, as long as people understand it 😝